libraries/harvest-queue/src/main/java/no/fdk/harvestqueue/HarvestQueue.java

```diff
@@ -0,0 +1,32 @@
+package no.fdk.harvestqueue;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.LinkedList;
+
+@Service
+public class HarvestQueue {
+    private static final Logger logger = LoggerFactory.getLogger(HarvestQueue.class);
+
+    private final LinkedList<QueuedTask> scheduledTasks = new LinkedList<QueuedTask>();
+
+
+    public void addTask(QueuedTask task) {
+        synchronized (scheduledTasks) {
+            if (scheduledTasks.contains(task)) {
+                logger.debug("Task already exists in queue: {}", task);
```

Comment:
jeg anbefaler litt mer eksplisitt:
logger.debug("Task already exists in queue, skipping: {}", task);

Reply:
Enig. Fixet.